lldb/test/API/lang/swift/parseable_interfaces/shared/TestSwiftInterfaceNoDebugInfo.py

```diff
@@ -129,7 +129,7 @@ def do_test(self):
         lldbutil.check_variable(self, child_y, False, value="0")
 
         child_x = var.GetChildMemberWithName("x") # MyPoint.x isn't public
```

Comment:
add `... but LLDB can find it through the type metadata`

Reply:
done